docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
+
+.. image:: images/relfrom.png
+   :width: 100%
+   :alt: alternate text
+
+
+Repeat these steps for each node class you wish to create. On this program, those would be: ``address.py``, ``intermediary.py``, ``officer.py``, and ``other.py``. You must add the following code to each of the files:
+
+address.py ::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipFrom
+    )
+
+    class Address(StructuredNode):
+        sourceID       = StringProperty()
+        country_codes  = StringProperty()
+        valid_until    = StringProperty()
+        address        = StringProperty()
+        countries      = StringProperty()
+        node_id        = StringProperty()
+        officers       = RelationshipFrom('.officer.Officer', 'REGISTERED_ADDRESS')
+        intermediaries = RelationshipFrom('.intermediary.Intermediary', 'REGISTERED_ADDRESS')   
+
+
+intermediary.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo
+    )
+
+    class Intermediary(StructuredNode, NodeUtils):
+        sourceID      = StringProperty()
+        valid_until   = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        status        = StringProperty()
+        entities      = RelationshipTo('.entity.Entity', 'INTERMEDIARY_OF')
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+
+
+officer.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Officer(StructuredNode):
+        sourceID      = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        valid_until   = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+        entities      = RelationshipTo('.entity.Entity', 'OFFICER_OF')
+
+other.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Other(StructuredNode):
+        sourceID    = StringProperty()
+        name        = StringProperty()
+        valid_until = StringProperty()
+        node_id     = StringProperty()
+        countries   = StringProperty()
+        addresses   = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+Modify the __init__.py file:
+==================================
+Now we need to add a little bit of code to the __init__.py. If you noticed in the previous code, we refer to other classes but we did no import them. e.g : ``.officer.Officer``. This means that it should not work because the program doesn't know what that is; Therefore, we are going to add all the imports to the init file so that each model has all the imports ::

```

Comment:
Actually Neomodel use the string ".officer.Officer" to import the model class `Officer` inside the module `.officer`. Re: https://github.com/neo4j-contrib/neomodel/blob/master/neomodel/relationship_manager.py#L364

If we leave the __init__.py empty, the models should be imported like this:
```from .models.officer import Officer```

The reason we import them on the __init__.py is to simplify the import statement. Now we can import them like this:
```from .models import Officer```

And now, we could also simplify the relationship definitions like:
```
        addresses     = RelationshipTo('Address', 'REGISTERED_ADDRESS')
        entities         = RelationshipTo('Entity', 'OFFICER_OF')
```